pkg/resources/database/utils.go

```diff
@@ -0,0 +1,77 @@
+package database
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/xeipuuv/gojsonschema"
+)
+
+var (
+	ServicesList = []string{
```

Comment:
This seems like something that doesn't belong into a file called utils.go

Reply:
Moved to `common.go`.